### DISCUSSION TITLE
We are thinking of changing our login page description, and you can contribute!

### DISCUSSION THREAD

**Discussion by shivankacker** [ohcnetwork/care_fe#2658] [Ideas]
Hey folks!
CARE has moved on from it's crucial role as a pandemic management system to something that can be used to manage health centers from the comfort of your devices with increased efficiency.
Currently, the login page only reflects CARE's initial goal, and we have to update it to something new that fits with CARE'S current use case.

![image](https://user-images.githubusercontent.com/23238460/172644125-454f3721-e615-4da5-8f84-1c8a350a3e1a.png)

So rather than sticking to one description, we can have multiple one liners that describe CARE which cycle every 5 seconds in a typewriter style.
Example -

>  A decentralized healthcare management tool

You can comment one liners you would like to add describing CARE on this thread, and relevant ones will be included.

**Comment by rithviknishad** [comment]
\+ Remote patient monitoring tool
